pkgs/development/python-modules/generic/default.nix

```diff
@@ -166,7 +116,8 @@ python.stdenv.mkDerivation (builtins.removeAttrs attrs ["disabled"] // {
        tmp_path=$(mktemp -d)
```

Comment:
This is out of scope for this refactoring. I'm introducing wheels, not trying to fix everything.